### ISSUE TITLE
logs: check for "unhandled errors"

### ISSUE THREAD

**Issue by pohly** [kubernetes/kubernetes#122005]
Once https://github.com/kubernetes/kubernetes/pull/121970 is merged and enough (all?) code is converted to `HandleErrorWithContext`, we can analyze logs from jobs like https://testgrid.k8s.io/sig-instrumentation-tests#kind-json-logging-master to find out:
- whether there are unhandled errors
- how many

With JSON output, jq might be enough to search for `logger=".*UnhandledError.*". See https://github.com/kubernetes/kubernetes/tree/master/test/integration/logs/benchmark#real-log-data for instructions on how to download those logs.

If there are none, then we might be able to add post-processing of all or at least some jobs and check that no such errors creep in. If there are some, we might need an allow list.

/cc @sttts

**Comment by rexagod** [comment]
@pohly https://github.com/kubernetes/kubernetes/pull/121970 is in, is there work left here (based on the description)?

**Comment by pohly** [comment]
Yes, we still lack the "and enough (all?) code is converted to HandleErrorWithContext" part before we can start with this issue. https://github.com/kubernetes/kubernetes/issues/126379 tracks that.
